pandas/_libs/tslibs/period.pyx

```diff
@@ -1246,6 +1246,37 @@ cdef class _Period(object):
 
     @property
     def dayofweek(self):
+        """
+        Return the day of the week.
+
+        This attribute returns the day of the week on which the particular
```

Comment:
If I'm not mistaken, this will return the day of the week on which the particular *starting* date for the given period occurs. I think this is important to mention, because at first I was confused thinking "how a period of one month can have a day of the week?".

Reply:
Yes, we have this discussion in another PR as well (https://github.com/pandas-dev/pandas/pull/20277/files#r173651389). Only, it is not necessarily the starting date, it depends on the `freq`.

Reply:
Issue is here: https://github.com/pandas-dev/pandas/issues/20324.